deeds-dapp-webapp/src/main/webapp/skin/less/deeds.less

```diff
@@ -238,6 +239,24 @@ h4 {
 .primary-border-color {
   border: 1px solid #3f8487 !important;
 }
+
+.dark-grey-border-color {
+  border: 1px solid @darkGrey !important;
+}
+
+.logo-m {
+  filter: invert(47%) sepia(46%) saturate(419%) hue-rotate(134deg) brightness(89%) contrast(90%);
+  opacity: 0.5;
+  position: absolute;
+  right: -80px;
+}
+
+#btn-dark-grey:hover {
```

Comment:
id must be unique in page. Besides, why using a special effect that is different from standard UX of Meeds package ?

Reply:
It's new UX adopted for some buttons on dapp
Done!